network/src/connectivity_manager/test.rs

```diff
@@ -301,14 +318,12 @@ fn connect_to_seeds_on_startup() {
 
         // We expect the peer which changed its address to also disconnect.
         info!("Sending lost peer notification for seed peer at old address");
-        send_notification_await_delivery(
+        send_lost_peer_await_delivery(
```

Comment:
I wonder if the "await_delivery" is kind of implicit in the fact that this is an awaitable / async?

Reply:
Yeah, I wasn't a fan of the "await_delivery", but I kept it the way it was